enterprise/frontend/src/metabase-enterprise/sandboxes/selectors.ts

```diff
@@ -1,4 +1,5 @@
 import { createSelector } from "@reduxjs/toolkit";
+import _ from "underscore";
```

Comment:
I think this is unused now? I think `_` gets an exception from unused imports rule